app/views/nd_certificates/certificate.html.erb

```diff
@@ -13,6 +13,6 @@
       Esta certidão é válida por 30 (trinta) dias a contar da data de sua emissão e pode ser utilizada para comprovar a regularidade financeira da unidade perante quaisquer instituições ou interessados.
     </p>
 
-    <p class="lead mt-5"><strong>Data de emissão:</strong> <%= I18n.l(@nd_certificate.issue_date) %></p>
+    <p class="lead mt-5"><strong>Data de emissão:</strong> <%= I18n.l(@nd_certificate.issue_date.to_date) %></p>
```

Comment:
No certificado é pedido a exibição da hora de emissão, se colocar o to_date ele só vai exibir a data e não a hora. Referente à correção dos testes que quebra as vezes foi feito no PR de certficado de DN.

Reply:
Massa Arthur! Vou puxar a mesma solução pra esse arquivo então pra que não quebre o CI. Faço manualmente ou via merge conflict dependendo de qual branch for primeiro pra main.